src/Map.js

```diff
@@ -226,6 +233,62 @@ export default class Map extends MapComponent<LeafletElement, Props> {
       }
     }
 
+    if (boxZoom !== fromProps.boxZoom) {
+      if (boxZoom === true) {
+        this.leafletElement.boxZoom.enable()
+      } else {
+        this.leafletElement.boxZoom.disable()
+      }
+    }
+
+    if (doubleClickZoom !== fromProps.doubleClickZoom) {
+      if (doubleClickZoom === true) {
+        this.leafletElement.doubleClickZoom.enable()
+      } else {
+        this.leafletElement.doubleClickZoom.disable()
+      }
+    }
+
+    if (dragging !== fromProps.dragging) {
+      if (dragging === true) {
+        this.leafletElement.dragging.enable()
+      } else {
+        this.leafletElement.dragging.disable()
+      }
+    }
+
+    if (keyboard !== fromProps.keyboard) {
+      if (keyboard === true) {
+        this.leafletElement.keyboard.enable()
+      } else {
+        this.leafletElement.keyboard.disable()
+      }
+    }
+
+    if (scrollWheelZoom !== fromProps.scrollWheelZoom) {
+      if (scrollWheelZoom === true || typeof scrollWheelZoom === 'string') {
+        this.leafletElement.scrollWheelZoom.enable()
+      } else {
+        this.leafletElement.scrollWheelZoom.disable()
+      }
+    }
+
+    if (tap !== fromProps.tap) {
+      if (tap === true) {
+        this.leafletElement.tap.enable()
+      } else {
+        this.leafletElement.tap.disable()
+      }
+    }
+
+    if (touchZoom !== fromProps.touchZoom) {
+      if (touchZoom === true || typeof touchZoom === 'string') {
```

Comment:
Why checking if `touchZoom` is a string?

Reply:
As above - [link](http://leafletjs.com/reference-1.3.0.html#map-touchzoom)